src/features/structureProvider.ts

```diff
@@ -0,0 +1,23 @@
+import { FoldingRangeProvider, TextDocument, FoldingContext, CancellationToken, FoldingRange } from "vscode";
+import AbstractSupport from './abstractProvider';
+import { blockStructure } from "../omnisharp/utils";
+import { Request } from "../omnisharp/protocol";
+
+export class structureProvider extends AbstractSupport implements FoldingRangeProvider
```

Comment:
Should this be in capital ?

Reply:
I agree

Reply:
Will fix. Also--it looks like many other classes start with Capital letters but have filenames that start with lowercase letters. Should we try to make this consistent or is this some typescriptism?